lib/oban/plugins/cron.ex

```diff
@@ -107,9 +122,32 @@ defmodule Oban.Plugins.Cron do
   def handle_info(:evaluate, %State{} = state) do
     state = schedule_evaluate(state)
 
-    Query.with_xact_lock(state.conf, state.lock_key, fn ->
-      insert_jobs(state.conf, state.crontab, state.timezone)
-    end)
+    start_metadata = %{
+      config: state.conf,
+      crontab: state.crontab,
+      name: state.name,
+      interval: state.interval
```

Comment:
The `interval` is only configurable for testing and the module should only be looked up through the registry. I don't think the `name` or `interval` should be included, and the `crontab` is kind of questionable as well.

Reply:
Sounds good. I wasn't sure if that information was useful or not. I'll remove the 3 you mentioned and add the `:plugin` field instead.